app/views/users/index.html.haml

```diff
@@ -5,11 +5,13 @@
   %ul.nav.nav-tabs
     %li{class: (params[:type].blank? && params[:role].blank?) && 'active' }
       = link_to 'All', users_path
-    %li{class: params[:role] == 'none' && 'active' }
-      = link_to 'New', users_path(role: 'none')
+    %li{class: params[:type] == 'without_role' && 'active' }
```

Comment:
Better enclose in parentheses, to reduce confusion

    class: (params[:type] == 'without_role') && 'active'